src/screens/Asl/components/AslTable/index.js

```diff
@@ -56,6 +57,83 @@ const AslTable = props => {
     const [showVideo, setShowVideo] = useState(false);
     const [videoUrl, setVideoUrl] = useState('');
 
+    const wrapStyle = {
+      left: 100,
+      top: 100,
+      width: 500,
+      height: 500
+    }
+    const [style, setStyle] = useState({
+        left: 100,
+        top: 100,
+        width: 400,
+        height: 200
+    })
+    // 初始数据， 因为不需要重新render 所以用 useRef
```

Comment:
I've never seen useRef be used to hold small constants before; can you add a reference (URL) where this is best or recommended practice?